ee/experimental/incident_utils.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+import pandas as pd
+import networkx as nx
+
+from typing import List
+
+from keep.api.models.db.alert import Alert
+
+
+def mine_incidents(alerts: List[Alert], incident_sliding_window_size: int=6*24*60*60, statistic_sliding_window_size: int=60*60, 
+                   jaccard_threshold: float=0.0, fingerprint_threshold: int=1):
+    """
+        Mine incidents from alerts.
```

Comment:
add docs on whats/how the parameters affect the results? when would I want to change one of them? @GlebBerjoskin

Reply:
+ I think all docs about EE should be internal by default